### DISCUSSION TITLE
Zikula 4 status

### DISCUSSION THREAD

**Discussion by KratorD** [zikula/core#5340] [General]
Hello.

I've read the readme and know that Zikula development is more or less dormant. This is just out of curiosity, what is the current status of Zikula 4? What has it developed?
(If you wish, you can use this discussion to report milestones)

Thanks and greetings.

**Comment by Guite** [comment]
I don't have time to write much here at the moment, but the changelog shows the most important changes at a glance:

https://github.com/zikula/core/blob/main/CHANGELOG-4.0.md

Most interesting part here is the _Features_  section apparently, as this points out some replacements where we exchanged old self-made stuff in favour of existing vendor bundles.

Its worth to take a look at these docs:

- https://symfony.com/bundles/EasyAdminBundle/4.x/index.html
- https://docs.nucleos.rocks/projects/user-bundle/en/stable/
- https://docs.nucleos.rocks/projects/profile-bundle/en/stable/

In addition to the changelog, there are some other minor innovations, like the `BundleInitializer` concept that allows for executing custom init code after a bundle has been installed and configured with Symfony Flex. Such things are not well tested yet though.

**Reply by ThiloWitt** [reply]
Is there a beta version that we can test, even if only partially?

**Reply by Guite** [reply]
Not yet, because the initial creation of admin credentials is missing.

In theory you could try to checkout the current source code, run `composer install` and follow the NucleosUserBundle docs in order to create a super admin. See https://docs.nucleos.rocks/projects/user-bundle/en/stable/command_line_tools/ to get started.

You would need to imitate this: https://github.com/zikula/core/blob/main/src/system/UsersBundle/Bundle/Initializer/UsersInitializer.php

No idea if this will be enough though.
